scripts/scil_apply_transform_to_tractogram.py

```diff
@@ -2,13 +2,22 @@
 # -*- coding: utf-8 -*-
 
 """
-    Transform tractogram using an affine/rigid transformation.
-
-    For more information on how to use the various registration scripts
-    see the doc/tractogram_registration.md readme file
+Transform tractogram using an affine/rigid transformation.
+
+For more information on how to use the various registration scripts
+see the doc/tractogram_registration.md readme file
```

Comment:
Can you please update this file. It seems out-dated with current naming conventions. I have raised an issue on this. You could close the issue at the same time.